src/advanced_inv.h

```diff
@@ -336,7 +351,13 @@ class advanced_inventory
 
         bool exit;
 
+        // store/load settings (such as index, filter, etc)
+        void save_settings(bool only_panes = false);
+        void load_settings();
+        void do_return_entry();
+
         static std::string get_sortname(advanced_inv_sortby sortby);
+        // if called with default arg, will exit after first completion (if not AIM_ALL)
```

Comment:
I don't get the comment. The function has no parameters at all. There is no default argument either.